README.md

```diff
@@ -0,0 +1,224 @@
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <h3><b>Hello World README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Fast Lane Reservation Backend <a name="about-project"></a>
+
+An app to book a race care to test in the track.
+[Link to the frontend](https://github.com/Khaled-AlKhateeb/fast-lane-reservations_frontend)
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Rails API:** The endpoint generate a random greeting message.
+- **Redux:** Makes the data fetching for the endpoint API and stores the state of the app.
+- **React Components:** Displays the greeting message with the greeting component.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A Terminal.
+- A web browser.
+- A code editor.
+- Ruby,Rails and Postgres Should be installed in your machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Khaled-AlKhateeb/fast-lane-reservation_backend.git
+```
+
+### Install
+
+Install this project with:
+
+- First install the ruby gems:
+
+```sh
+  cd fast-lane-reservation_backend
+  bundle install
+```
+
+- Then Install the node packages by:
+
+```sh
+  yarn install
+```
+
+> OR:
+
+```sh
+  npm install
+```
+
+- Create the database:
+
+```sh
+  rails db:create
+  rails db:migrate
+  rails db:seed
```

Comment:
- [ ] You are including in your instructions `rails db:seed`, however, the `seeds.rb` file is empty at the time of this review.  Kindly, either add some content to `seeds.rb` to populate the database, or remove this `rails db:seed` instruction from the readme to prevent confusion.